pkg/util/scrubber/scrubber.go

```diff
@@ -118,7 +118,9 @@ func (c *Scrubber) scrubReader(file io.Reader) ([]byte, error) {
 	first := true
 	for scanner.Scan() {
 		b := scanner.Bytes()
-		if !commentRegex.Match(b) && !blankRegex.Match(b) && string(b) != "" {
+		if blankRegex.Match(b) || string(b) == "" {
```

Comment:
I _think_ we don't need the `string(b)` validation - will check and remove if need be. Otherwise the `blankRegex` can probably be improved to do that with no allocations.